content/lessons/chapter-1/outro-2.tsx

```diff
@@ -6,7 +6,7 @@ import { ChapterEnd } from 'ui'
 export const metadata = {
   title: 'chapter_one.outro_one.title',
   image: '/assets/images/chapter-1-finish.jpg',
-  theme: 'bg-transparent',
+  theme: 'bg-transparent/20',
```

Comment:
good point, what do you think we should choose for a bg-color of the menu when dealing with a tansparent navbar?
Just the standard 'bg-back'?

Reply:
Standard bg-back will work well here.

1. It has looked good on the pages preceding it.
2. It will maintain the theme consistency with the rest of the chapter.